node_utils.js

```diff
@@ -38,10 +38,12 @@ function get_conf() {
 	return conf;
 }
 
-function get_redis_subscriber(kind = "redis_queue", options = {}) {
+async function get_redis_subscriber(kind = "redis_queue", options = {}) {
 	const conf = get_conf();
 	const host = conf[kind] || conf.redis_async_broker_port;
-	return redis.createClient({ url: host, ...options });
+	const client = redis.createClient({ url: host, ...options });
+	await client.connect();
```

Comment:
This breaks build when redis isn't available. Try `bench build` without redis service running. 

```
Error: connect ECONNREFUSED 127.0.0.1:13000
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1278:16) {
  errno: -111,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 13000
}
error Command failed with exit code 1.
```

Reply:
Why await connection?

Reply:
https://github.com/frappe/frappe/pull/20925/commits/65d83b2709fb9fe975cbc8b87ea3ca60271f728c brings back:

![image](https://github.com/frappe/frappe/assets/7548295/d37e1aa7-201f-4ab7-99cd-4bafdc7fa5ad)